doc/release/technotes/README.extern.rst

```diff
@@ -85,11 +86,13 @@ To help with mapping types between C and Chapel, Chapel installations
 contain a standard module named 'SysCTypes' (located under
 $CHPL_HOME/modules/standard/gen/...).  This module defines a number of
```

Comment:
I'd do something like ``$CHPL_HOME/modules/standard/gen/`` - would we do a cross-reference to SysCTypes documentation later?